trino/dbapi.py

```diff
@@ -280,32 +298,32 @@ def __init__(self, connection, request, legacy_primitive_types: bool = False):
         self._request = request
 
         self.arraysize = 1
-        self._iterator = None
-        self._query = None
+        self._iterator: Optional[Iterator[Any]] = None
```

Comment:
Isn't this an `Iterator[List[Any}}`?

Reply:
Changed to `Optional[Iterator[List[Any]]]`